db/patterns/lotustar.eno

```diff
@@ -0,0 +1,14 @@
+name: Lotustar
+category: advertising
```

Comment:
Potentially, "customer_interaction" could also be a fit for an entertainment system, even if it includes ads.

Reply:
updated in 504c8df43e55b24fc35080f9fb14e328b3731263